packages-content-model/roosterjs-content-model-editor/lib/publicApi/table/editTable.ts

```diff
@@ -30,9 +32,12 @@ import {
  * @param editor The editor instance
  * @param operation The table operation to apply
  */
-export default function editTable(editor: IContentModelEditor, operation: TableOperation) {
+export default function editTable(
+    editor: IContentModelEditor,
+    operation: TableOperation,
+    border?: Border
```

Comment:
All other editing operations don't have parameter, but border needs parameter. So suggest split border into a different API, no need to merge into editTable.

Reply:
Similar with set table cell shade color.